### DISCUSSION TITLE
Arturia Beatstep jumps to some random parameter value

### DISCUSSION THREAD

**Discussion by Krampik** [rsjaffe/MIDI2LR#561] [Problems]
Does anyone have Arturia Beatstep? I have a problem with LR (11.0.1) and the MIDI2LR plugin (5.1.1.0) of course. When turning the encoders, very often, instead of changing the current value, it jumps to some random parameter value. For example, I have 4800K, turn it to the left by 1 click and the value is, for example, 3182K and then it works normally. I change the value by those over 100K left or right. This happens with every parameter, randomly, independently. This is obviously very annoying and instead of speeding up the processing of photos, it slows them down. Anyone had a similar problem and somehow solved it?

**Comment by rsjaffe** [comment]
Do you use layers? See https://github.com/rsjaffe/MIDI2LR/wiki/MIDI-Controller-Setup#problems-with-control-surfaces-that-have-layers. The rest of that wiki page has other potentially useful information.

Also, is pickup mode off? That could contribute to the problem. See https://github.com/rsjaffe/MIDI2LR/wiki/Application-Settings-Dialog .

**Reply by Krampik** [reply]
I have layers and disable pickup mode, but when i use one layer its the same problem

**Reply by rsjaffe** [reply]
Enable pickup mode to get rid of the jumping. I don't know why you're having sync problems with the controller--I suggest reading the wiki page on MIDI controller setup to see if that helps you.

**Reply by Krampik** [reply]
ok. I checked exactly what it was. The CC values are not syncronized at all when you go to the next photo, hence the jumps. It also does not synchronize when in the same photo I change the value by clicking the slider with the mouse and then turning the encoder, it returns to the value from the encoder. It is as if the encoders behave like potentiometers ...

**Reply by rsjaffe** [reply]
When not in pickup mode, MIDI2LR does not attempt to synchronize the MIDI controller with the picture's settings. I did that because I thought people would choose non-pickup mode when they don't want the controllers to be overridden by the picture's settings. I can change that behavior, but I believe the current behavior is how people using non-pickup mode want it to work.

**Reply by Krampik** [reply]
i don't know what i can do. I plugged in my old X-touch mini controller and it works perfectly fine, without any interference with the settings, and this one won't.

**Reply by JaimeTak** [reply]
@Krampik, did you get it working? Question: do all Beatstep buttons o the left side are mappable on midi2lr?

**Comment by Krampik** [comment]
yes I managed to get it to work properly. I see only button stop can be mappable. and offcourse big knob

**Reply by JaimeTak** [reply]
Thanks

**Reply by henjo10** [reply]
> yes I managed to get it to work properly. I see only button stop can be mappable. and offcourse big knob

@Krampik did you manage to solve the synchronization of the values between photos? I have the same problem between photos. It keeps the previous value and does not reset between photos. If you could explain how you did it, I would appreciate it
